### PR TITLE
Bring the app content in line with the prototype

### PR DESCRIPTION
**Pull request by peteryates**
Prior to MVP we need to bring this app inline with the prototype.

Here's the list of things to fix from @Bethlt4 

Do now:

* [x] Fix the title on the homepage
* [x] Fix the content on the splash page
* [x] Do exact copy from prototype to app of the CCP index page (key stage content/rationale page)
* [x] Do exact copy from prototype to app of show CCP page (year 7 history curriculum)
* [x] Do exact copy from prototype to app of show unit page (Medieval realms unit)- keep two divisions of data: move benefits up
* [x] Number the tabs
* [x] Lesson plan- Battle of Hastings is Lesson 3 not 1 - we have 2 full lessons that can currently be added to the app
* [x] Do exact copy from prototype to app of knowledge overview tab - add the lesson structure guidance (rationale) 
* [x] Lesson contents tab (and all tabs), include text headers 
* [x] Downloads tab - remove the word ‘plan’ from the last sentence

-----

## Next sprint, after some planning...

* [ ]  **design task:** for changing radio buttons for year groups?
* [ ]  **dev task:** change rationale paragraphs in app to be divided into 2 sections that reflect supplier upload e.g. rationale, theory, planning, guidance (how to teach)?
* [ ] **separate task:** when a supplier doesn’t provide a piece of data e.g. ‘core knowledge’, needs to still be displayed but something like ‘not applicable’ to go underneath (might need a ‘not applicable’ option for certain fields in the supplier upload)

------

# Changes

| Prototype | Rails app |
| ----- | ----- |
| Screenshot from 2020-03-13 14-00-40 | Screenshot from 2020-03-13 14-00-42 |
| Screenshot from 2020-03-13 14-03-42 | Screenshot from 2020-03-13 14-03-44 |
| Screenshot from 2020-03-13 14-04-58 | Screenshot from 2020-03-13 14-04-59 || Screenshot from 2020-03-13 14-05-47 | Screenshot from 2020-03-13 14-05-49 |
| Screenshot from 2020-03-13 14-07-14 | Screenshot from 2020-03-13 14-07-16 |
| Screenshot from 2020-03-13 14-08-20 | Screenshot from 2020-03-13 14-08-22 |
| Screenshot from 2020-03-13 14-08-56 | Screenshot from 2020-03-13 14-08-55 |
| Screenshot from 2020-03-13 14-10-26 | Screenshot from 2020-03-13 14-10-28 |

### CODE REVIEW

File: app/views/teachers/units/show.slim [hunk 1]
```diff
@@ -5,17 +5,15 @@
...
-
     section.benefits
-      h2.govuk-heading-m
-        | How this lesson sequence benefits pupils’ learning
-
       p = markdown(@unit.benefits)
 
   .govuk-grid-column-one-third
-    = render 'siblings', units: @unit.complete_curriculum_programme.units
+    = render 'siblings', units: @unit.complete_curriculum_programme.units.ordered_by_position
```
Comment (rjlynch) [inline app/views/teachers/units/show.slim]: We should probably disable the links to other units if they have no content (like on the year show view)

File: app/webpacker/styles/_lessons.scss [hunk 2]
```diff
@@ -1,10 +1,42 @@
...
+
+    .govuk-tabs__list-item {
+      &::marker {
+        font-weight: bold;
+      }
+
+      // the design system formats this tab list with a long dash "\2014" when narrower than
+      // tablet. As we're unconventionally using a ol rather than ul, remove the left padding
+      // the dash appears *after* the number but before the list item content
+      &::before {
```
Comment (rjlynch) [inline app/webpacker/styles/_lessons.scss:34]: Nice

File: app/webpacker/styles/_units.scss [hunk 3]
```diff
@@ -1,21 +1,30 @@
 // units table
```
Comment (rjlynch) [inline app/webpacker/styles/_units.scss:1]: Not sure where to add this but it's todo with the unit page.
When viewing the unit#show page on a small screen the side bar breaks above the lesson table which seems a bit odd
Screenshot 2020-03-13 at 16 42 38

**Review by rjlynch — APPROVED** [review]
LGTM some minor comments.
I'm going to have to self isolate to get the merge conflicts in db/seeds resolved when my feature branch is done! 😿

**Comment by rjlynch** [comment]
Heroku will need reseeding once this is merged

File: app/views/teachers/units/show.slim (hunk 1 above)
Comment (peteryates) [reply app/views/teachers/units/show.slim]: Yeah, well spotted :owl:

File: app/webpacker/styles/_units.scss (hunk 3 above)
Comment (peteryates) [reply app/webpacker/styles/_units.scss:1]: Ah yeah, I tried to make the table wider by placing it in a separate row underneath. I'll move it up to the same column as the text content and it should flow in the right order :+1:

**Pull request merged**